app/models/features_plan.rb

```diff
@@ -1,6 +1,4 @@
 class FeaturesPlan < ApplicationRecord
-  self.primary_key = nil # this column does not exist, but rails needs to have something
```

Comment:
This prevents some automatic rails detection of composite primary keys. After removing this, `FeaturesPlan.primary_key` still returns `nil` and prints a warning the primary key is ignored as ActiveRecord doesn't support composite primary keys.

I think having like this is better because there is some work in Rails to support composite keys which may later kick-in automatically. No need to force `nil` here.